src/FieldDescription/FieldDescriptionInterface.php

```diff
@@ -53,7 +53,7 @@
  *  translation_domain?: string,
  *  type?: string,
  *  virtual_field?: bool
- * }&array<string, mixed>|array<empty, empty>
+ * }|array<empty, empty>
```

Comment:
I don't know if it's de proper fix, but phpstan does not complain

Reply:
This is not valid IMHO, and we'll get some others errors.
Any option should be valid.

For instance, in my code, I'm using
```
->add('clients', null, [
    'template' => 'admin/sonata/list/contract_clients.html.twig',
    'link'     => 'admin_assurancevie_client_edit',
])
```

Reply:
Maybe the issue should be reported to phpstan